decidim-generators/lib/decidim/generators/app_templates/bullet_initializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+if defined?(Bullet)
+  Rails.application.config.after_initialize do
+    Bullet.enable = true
+    Bullet.bullet_logger = true
+    Bullet.rails_logger = true
+    Bullet.stacktrace_includes = %w(
+      decidim-accountability
```

Comment:
can you check if something like "decidim-*" will work?

Reply:
I'll give it a try. I tried getting the list of gems from `Decidim::GemManager` using `#plugins` by I didn't succeed and felt too much for this PR. Would be `# worth anywhere else to deserve some more investigation?

Reply:
nevermind. The last commit did the trick so we don't need to touch `GemManager` now.